go.mod

```diff
@@ -185,6 +185,7 @@ require (
 	gomodules.xyz/jsonpatch/v3 v3.0.1
 	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c
 	google.golang.org/grpc v1.38.0
+	google.golang.org/protobuf v1.26.0
```

Comment:
I am not sure if we can make this leap, if I recall correctly we already have some pins to older versions that were required by grpc, and in turn, those were forced by an etcd library (`go.etcd.io/etcd/client/v2`).

Reply:
The updated version of this package, in Molecule, has no (non-test) dependencies, so this should be fine.